### PR TITLE
Method for installing ESPIDF offline

### PR DESCRIPTION
**Pull request by liqijian101**
Method for installing ESPIDF offline

### CODE REVIEW

File: src/content/esp3d-tft/v1.x/installation/_index.md [hunk 1]
```diff
@@ -14,6 +14,8 @@ weight = 2
 * Update the project submodules   
 `git submodule update --init --recursive`
 
+If you need to climb over a wall to connect to GitHub, it is recommended to use dev sidecar.
```
Comment (luc-github) [inline src/content/esp3d-tft/v1.x/installation/_index.md:17]: @liqijian101 what do you mean ?

**Review by luc-github — COMMENTED** [review]
@liqijian101 Can you answer my question or elaborate more ?

File: src/content/esp3d-tft/v1.x/installation/_index.md (hunk 1 above)
Comment (liqijian101) [reply src/content/esp3d-tft/v1.x/installation/_index.md:17]: 在中国，有时不能顺畅的连接到github，需要使用网络代理功能，因此我推荐使用dev sidecar辅助软件。它可以帮助连接到github.

In China, sometimes you can't connect to github smoothly and need to use the network proxy function, so I recommend using the Dev sidecar soft. It helps connect to github.

Comment (luc-github) [reply src/content/esp3d-tft/v1.x/installation/_index.md:17]: Ok I understand now, do you have a link of this software for reference ?

Comment (luc-github) [reply src/content/esp3d-tft/v1.x/installation/_index.md:17]: is this one ? https://github.com/docmirror/dev-sidecar

Comment (liqijian101) [reply src/content/esp3d-tft/v1.x/installation/_index.md:17]: Yes

**Comment by luc-github** [comment]
Merged - thank you,
I will build site and upload it later today

**Pull request merged**